content/en/network_performance_monitoring/network_installation.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Network Performance Monitoring Installation
+kind: documentation
+disable_toc: true
+description: Collect your Network Data with the Agent.
+further_reading:
+- link: "https://www.datadoghq.com/blog/network-performance-monitoring"
+  tag: "Blog"
+  text: "Network Performance Monitoring"
+- link: "https://www.datadoghq.com/blog/monitoring-101-alerting/"
+  tag: "Blog"
+  text: "Monitoring 101: Alerting on what matters"
+---
+
+<div class="alert alert-warning">
+This feature is currently in beta. Request access by filling out the <a href="https://app.datadoghq.com/network/2019signup">Datadog Network Performance Monitoring Beta Request form</a>.
+</div>
+
+Network performance monitoring requires Datadog Agent v6.13+. To enable network performance monitoring, configure it in your [Agent main configuration file][1] based on your system setup:
+
+{{< tabs >}}
+{{% tab "Agent" %}}
+
+To enable network performance monitoring with the Datadog Agent, use the following configurations:
+
+1. Copy the system-probe example configuration:<br>
+`sudo -u dd-agent cp /etc/datadog-agent/system-probe.yaml.example /etc/datadog-agent/system-probe.yaml`
+2. Modify the system-probe configuration file to set the enable flag to `true`:<br>
+
+3. Optionally uncomment the `system_probe_config` parameter to add a custom object:
+    ```
+    ## @param system_probe_config - custom object - optional
+    ## (...)
+    #
+    system_probe_config:
+    ```
+
+4. Enter specific configurations for your System Probe data collection:
```

Comment:
Should `System Probe` be capitalized? If not, change it in all places.

Reply:
It should, capitalizing  everywherre